guides/release/components/built-in-components.md

```diff
@@ -21,9 +21,27 @@ Will become:
 <label for="site">Ember Question</label>
 <input id="site" type="text" value="How do text fields work?"/>
 ```
+### Naming with aria-label
 
-You can pass the following standard `<input>` attributes within the input
-helper:
+The `aria-label` property enables developers to name an element with a string that is not visually rendered.
+
+```handlebars
+<Input id="site" @value="How do text fields work?" aria-label="Ember Question"/>
+```
+
+The `aria-label` property is useful when there is no visible text content that will serve as an appropriate accessible name. 
+
+### Setting attributes on Input
+
+Just like a native `<input>` element, there are many different types of attributes that you can apply to Ember's `<Input />` component, such as `aria` attributes or `required`. 
+For example, the `aria-labelledby` property is useful for situations like a search input, where the search button can serve as the label for the input element:
+
+```handlebars
+<Input aria-labelledby="button-search" />
+<button id="button-search">Search</button>
+```
+
+Here are some other standard `<input>` attributes that are supported:
```

Comment:
Some of the lists just below are attributes and others are arguments (e.g. `@value`).

```suggestion
Here are some other standard `<input>` attributes and arguments that are supported:
```